.github/workflows/haskell.yml

```diff
@@ -44,39 +44,66 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [8.10.2+macOS, 8.8.3+inspection+no-fusion-plugin+Werror, 8.10+stack+lts-17, 8.6.5+no-fusion-flag, 8.6.5+streamk, 8.4.4+debug]
-        cabal_version: ["3.2"]
+        name:
+          - 9.4.5
+          - 9.2.8
+          - 8.10.2+macOS
+          - 8.8.3+inspection+no-fusion-plugin+Werror
+          - 8.10+stack+lts-17
+          - 8.6.5+no-fusion-flag
+          - 8.6.5+streamk
+          - 8.4.4+debug
         include:
+          - name: 9.4.5
+            ghc_version: 9.4.5
+            build: cabal
+            cabal_version: 3.8.1.0
+            disable_sdist_build: "y"
+            ignore_error: false
+            runner: ubuntu-latest
+          - name: 9.2.8
+            ghc_version: 9.2.8
+            build: cabal
+            cabal_version: 3.6.2.0
+            disable_sdist_build: "y"
+            ignore_error: false
+            runner: ubuntu-latest
           - name: 8.10.2+macOS
             ghc_version: 8.10.2
-            build: cabal-v2
```

Comment:
This should be `cabal-v2`. `cabal` defaults to `cabal-v1` as this version of packcheck (The CI test tool) is old.

Reply:
I was comparing the command used in the latest version of master but I'll switch to `cabal-v2`.